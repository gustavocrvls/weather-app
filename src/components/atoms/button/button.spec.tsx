import { Button } from '.'
import { createWithTheme } from '../../../specs/utils'

describe('Atoms/Button', () => {
  test('should match snapshot', () => {
    const component = createWithTheme(<Button>Button</Button>)

    expect(component.toJSON()).toMatchSnapshot()
  })
})
