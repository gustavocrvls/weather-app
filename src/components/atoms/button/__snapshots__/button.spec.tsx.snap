// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atoms/Button should match snapshot 1`] = `
<button
  className="sc-bdvvtL kWEXTY"
>
  Button
</button>
`;
